NN_start.js

```diff
@@ -17,13 +17,21 @@ nerthus.addon.consts.CDN_VERSION_SEPARATOR = '@'
 nerthus.addon.version = nerthus.addon.consts.MASTER
 nerthus.addon.version_separator = nerthus.addon.consts.CDN_VERSION_SEPARATOR
 nerthus.addon.filesPrefix = nerthus.addon.consts.CDN_PREFIX
-nerthus.addon.fileUrl = function(filename)
```

Comment:
po co to rozbijać na path i filename i wszędzie poprawiać?
wystarczy wstawić to jak mniemam i nie trzeba nic zmieniać w miejscach użycia i robić jakiś dziwnych checków na to co jest podane...

```
  filename = filename .split("/")
  filename [filename .length - 1] = encodeURIComponent(filename [filename .length - 1])
  filename = filename .join("/")
```

Reply:
encodeURIComponent() zmieni ci znak `/` na `%2F`
`encodeURIComponent("/path/smth");`
zwraca `"%2Fpath%2Fsmth"`

Ale jak teraz poszukałem trochę to istnieje też funkcja `encodeURI` która będzie chyba działać dokładnie jak tego potrzeba, zaraz zmienię.